src/index.html

```diff
@@ -11,17 +11,17 @@
       <h1 id='header-name'>FITLIT</h1>
       <button aria-label='profile' id='profile-button'></button>
       <section class='hide' id='user-info-dropdown'>
-        <h5 id='dropdown-name'></h5>
+        <!-- <h5 id='dropdown-name'></h5>
         <p class='dropdown-p' id='dropdown-email'></p>
         <p class='dropdown-p' id='dropdown-goal'></p>
         <br/>
         <p class='dropdown-p' id='dropdown-friends'>FRIENDS TOTAL STEPS THIS WEEK</p>
-        <section id='dropdown-friends-steps-container'></section>
+        <section id='dropdown-friends-steps-container'></section> -->
       </section>
     </header>
     <main>
-      <section class='card-container' id='steps-card-container'>
-        <section class='main-card' id='steps-main-card'>
+      <section class='steps card-container' id='steps-card-container'>
```

Comment:
should we look into generating the main cards through the dom?

Reply:
maybe after we move DOM manipulation related code to a different file?

Reply:
We could do that - but do you think that would create any issues populating the data on the main page in our fetch requests?